app/controllers/peoplefinder/search_controller.rb

```diff
@@ -1,6 +1,7 @@
 module Peoplefinder
   class SearchController < ApplicationController
     def index
+      @search = true
```

Comment:
I don't think the name `@search` communicates very clearly what it's precise role is.